bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/thing/server-thing-type.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="freeboxos"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="revolution">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="api"/>
+		</supported-bridge-type-refs>
+
+		<label>Freebox Revolution</label>
+		<description>Provides various informations regarding the status of the Freebox Revolution Server</description>
+
+		<channel-groups>
+			<channel-group typeId="display" id="display"/>
+			<channel-group typeId="sensors" id="sensors"/>
+			<channel-group typeId="samba" id="samba"/>
+			<channel-group typeId="sysinfo" id="sysinfo"/>
+			<channel-group typeId="actions" id="actions"/>
+			<channel-group typeId="connection-status" id="connection-status"/>
+			<channel-group typeId="connectivity" id="connectivity"/>
+		</channel-groups>
+
+		<representation-property>hostAddress</representation-property>
```

Comment:
Normally, you should refer a configuration parameter here. I don't find the parameter "hostAddress".